### PR TITLE
Code Style: Use Implicit Usings; Be Consistent in Usings

### PR DESCRIPTION
**Pull request by viceroypenguin**

### CODE REVIEW

**Review by MaceWindu — APPROVED** [review]

**Comment by MaceWindu** [comment]
btw, don't merge it yet - I would like to run tests (after https://github.com/linq2db/linq2db/pull/3730 merged)

File: Examples/Remote/Grpc/Client/Client.csproj [hunk 1]
```diff
@@ -30,4 +30,8 @@
 		</Compile>
 	</ItemGroup>
 
+	<ItemGroup>
```
[the remark this answers is not in the document]
Comment (viceroypenguin) [reply Examples/Remote/Grpc/Client/Client.csproj:33]: Unfortunately, the `ImplicitUsings` source generator does not review `Directory.Build.props` for this information, so it has to be added to individual projects instead.

**Comment by igor-tkachev** [comment]
We need to revert this PR.

**Comment by viceroypenguin** [comment]
> We need to revert this PR.

Is there a bug with this PR?

**Comment by viceroypenguin** [comment]
More importantly, why was it reverted without a PR to discuss?

**Comment by igor-tkachev** [comment]
If you want to make global changes like this, firstly you need to explain benefits we can get. If we decide that the benefits are considerable, you have to go through all unfinished PRs and check how much work we need to do to merge your changes. 

If "benefits" - "work to do" is still positive value then we can approve this kind of PR.

**Pull request merged**